portfolio/src/main/webapp/favorite-quotes.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Ariel Salazar: Portfolio Page</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body onload="getQuote()">
```

Comment:
Instead of running the js via html, can you do this fully in JS? (See "DOMContentLoaded")